src/main/resources/templates/placeOrder.html

```diff
@@ -17,121 +17,121 @@
+                        <option value="WV">West Virginia</option>
+                        <option value="WI">Wisconsin</option>
+                        <option value="WY">Wyoming</option>
+                    </select>
+                </div>
+                <div class="form-group col-md-2">
+                    <label for="inputZip">Zip</label>
+                    <input type="text" class="form-control" id="inputZip" th:field="*{customerZip}">
+                </div>
             </div>
-        </div>
-        <div class="form-row">
-            <div class="form-group col-md-6">
-                <label for="itemNumber">Add Item Number</label>
-                <input type="text" class="form-control" id="itemNumber" placeholder="Item number">
+            <div class="form-row">
+                <div class="form-group col-md-6">
+                    <label for="itemNumber">Add Item Number</label>
+                    <input type="text" class="form-control" id="itemNumber" placeholder="Item number" th:field="*{itemId}">
+                </div>
+                <div class="form-group col-md-6">
+                    <label for="quantity">Enter Quantity</label>
+                    <input type="text" class="form-control" id="quantity" placeholder="Quantity" th:field="*{quantity}">
+                </div>
             </div>
-            <div class="form-group col-md-6">
-                <label for="quantity">Enter Quantity</label>
-                <input type="text" class="form-control" id="quantity" placeholder="Quantity">
+            <div class="form-row">
+                <div class="form-group col-md-6">
+                    <label for="inlineFormCustomSelectPref">Shipping Vendor</label>
+                    <select class="custom-select my-1 mr-sm-2" id="inlineFormCustomSelectPref" th:field="*{carrier}">
+                        <option value="Choose" selected>Choose...</option>
+                        <option value="1">USPS</option>
+                        <option value="2">FedEx</option>
+                        <option value="3">UPS</option>
+                    </select>
+                </div>
             </div>
-        </div>
-        <div class="form-row">
-            <div class="form-group col-md-6">
-                <label for="inlineFormCustomSelectPref">Shipping Vendor</label>
-                <select class="custom-select my-1 mr-sm-2" id="inlineFormCustomSelectPref">
-                    <option selected>Choose...</option>
-                    <option value="1">USPS</option>
-                    <option value="2">FedEx</option>
-                    <option value="3">UPS</option>
-                </select>
+            <div class="form-row">
+                <div class="form-group col-md-6">
+                    <label for="itemNumber">Additional Notes</label>
+                    <input type="text" class="form-control" id="deliveryRequests" placeholder="Delivery Requests" th:field="*{deliveryRequest}">
```

Comment:
Added the field attribute to the input element. This will allow the element values to bind to the field of the object.